docs/references/api.rst

```diff
@@ -2178,36 +2427,62 @@ by origin.
 
         .. code-block:: bash
 
-           pro api u.pro.packages.summary.v1
+           pro api u.pro.services.dependencies.v1
 
       - Expected attributes in JSON structure:
 
-        .. code-block:: json
+        .. code-block:: js
 
-           {
-               "summary":{
-                   "num_installed_packages": 1,
-                   "num_esm_apps_packages": 2,
-                   "num_esm_infra_packages": 3,
-                   "num_main_packages": 4,
-                   "num_multiverse_packages": 5,
-                   "num_restricted_packages": 6,
-                   "num_third_party_packages": 7,
-                   "num_universe_packages": 8,
-                   "num_unknown_packages": 9,
-               },
-           }
+            {
+              "services": [
+                {
+                  "name": "one",
+                  "depends_on": [
+                    {
+                      "name": "zero",
+                      "reason": {
+                        "code": "one-and-zero",
+                        "title": "Service One requires service Zero."
+                      }
+                    },
+                    ...
+                  ],
+                  "incompatible_with": [
+                    {
+                      "name": "two",
+                      "reason": {
+                        "code": "one-and-two",
+                        "title": "Services One and Two don't go together."
```

Comment:
I realise this is a code rather than docs issue, but saying "don't go together" is quite idiomatic English and may not be well understood by those with English as a 2nd+ language. I think it's ok to say "are not compatible" or something similar.